### ISSUE TITLE
Unexpected behaviour when calling undefined object methods from a task on a Worker

### ISSUE THREAD

**Issue by dktapps** [krakjoe/pthreads#724]
*If you know how to write phpt tests, please open a PR with environment information, and your test, this will cause CI to test your issue before a human is able to look*

Environment
==========

  * PHP: PHP 7.2.0-dev (cli) (built: Jul 24 2017 10:49:45) ( ZTS MSVC15 (Visual C++ 2017) x64 )
  * pthreads: 3.1.7dev 634fc6d65301deb77ba267fc9e794ea33099e84c
  * OS: Windows 10 64-bit

Summary
======
Calling undefined object methods in a Task on a Worker without catching the resulting Error ought to cause a fatal error and stop the worker thread. Instead the task execution just does not continue and does not produce any output at all.

When using a try...catch it is shown that the Error is indeed being thrown in the task, but no output is produced. I expected that the Worker thread would crash, as with any other fatal error.

Reproducing Code
==============
```
<?php

class Task extends \Threaded{

	public function run(){
		$obj = new StoredObject();
		$obj->calc2(mt_rand(0, $i), mt_rand(0, $i));
	}
}

class StoredObject{
	private $func;

	public function __construct(){
		$this->func = function(int $a, int $b){
			return $a + $b;
		};
	}

	public function calc(int $a, int $b) : int{
		return $this->func($a, $b);
	}
}

//try it on main thread, this works fine and causes a fatal error due to uncaught error as expected
//$task = new Task();
//$task->run();

//on worker, produces no output at all
$worker = new \Worker();
$worker->start(PTHREADS_INHERIT_ALL);
$worker->stack(new Task());
while($worker->collect());
$worker->shutdown();
```
Expected Output
=============
```
Fatal error: Uncaught Error: Call to undefined method StoredObject::calc2() in C:\php\unexpected-test.php:8
Stack trace:
#0 C:\php\unexpected-test.php(33): Task->run()
#1 {main}
  thrown in C:\php\unexpected-test.php on line 8
```

Actual Output
============
Nothing whatsoever.

**Comment by krakjoe** [comment]
The `Worker` executes tasks in an isolated way by design: If a `Worker` were to crash when a `Threaded` object done something stupid, it would make `Worker` threads quite unpredictable.

tl;dr this is totally expected
